pyproject.toml

```
@@ -0,0 +1,27 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "fool-ai-detector"
+version = "0.0.1"
+authors = [
+    { name="Raphael Weber"},
+    { name="Max Buchholz"},
+    { name="Lukas Müller"},
+    { name="Levi Otterbach"},
```

Last comma is unnecessary
```suggestion
    { name="Levi Otterbach"}
```

---

I deleted it but now I'm pretty sure that's the syntax so I added it again.

---

What do you mean? That the comma is required?

---

I am pretty sure that after the last element of an array, there should be no further comma, so please keep it as it is. See also: https://toml.io/en/v1.0.0#array